docs/en/recommended_topics/deploy/mmdeploy_guide.md

```diff
@@ -19,6 +19,14 @@ Currently our deployment kit supports on the following models and backends:
 
 Note: ncnn and other inference backends support are coming soon.
 
+## Installation
+
+Please install mmdeploy by following [this](https://mmdeploy.readthedocs.io/en/latest/get_started.html) guide.
+
+```{note}
+If you install mmdeploy prebuilt package, please also clone its repository by 'git clone https://github.com/open-mmlab/mmdeploy.git --depth=1' to get the deployment config files.
```

Comment:
actually, mmyolo has its own deploy configs. So to clone mmdeploy is to get tools.

Reply:
fix typo and remove `\` in line 98 
```
\`YOLOv5\` to \`ONNXRuntim\`
```

Reply:
fix typo in line 56 `nums` -> `nms`